linalg/hypre.cpp

```diff
@@ -33,7 +33,7 @@ namespace mfem
 
 Hypre::Hypre()
 {
-#if MFEM_HYPRE_VERSION >= 21500
+#if MFEM_HYPRE_VERSION >= 21900
```

Comment:
`HYPRE_Init` is available in hypre v2.15.0, it just has arguments: https://github.com/hypre-space/hypre/blob/v2.15.0/src/utilities/_hypre_utilities.h#L1767 -- we can just use `(0, NULL)` for versions >= 2.15.0 and < 2.19.0.

Reply:
We can, but I think it is also fine not to leave it as is. We did not support `HYPRE_Init` until recently, so it is fine if we don't support it for versions  >= 2.15.0 and < 2.19.0

Reply:
Then this version check should be used also for `HYPRE_Finalize()`: https://github.com/mfem/mfem/blob/c7c4e7ce3a585f1b1b7ffe71d31452658fc728ad/linalg/hypre.cpp#L50-L52.

Reply:
Done